hw/top_englishbreakfast/util/prepare_sw.py

```diff
@@ -124,21 +124,6 @@ def main():
         with open(path_out + file_name_new, "w") as file_out:
             file_out.write(text)
 
-    # 4.
```

Comment:
Could you please also remove Lines 80 and 81 of this file? These comments outline why this block No. 4 here is needed and what it does.